### PR TITLE
add postel's law SHOULD to the eip-55 paragraph per implementer feedback

### PR DESCRIPTION
**Pull request by bumblefudge**
Per discussion with implementer @chris13524 and @pedrouid , I added an explicit Postel's Law SHOULD to the eip-55 checksum-case section of EIP155/caip10.md.  Not breaking in any way so 2 reviews from any interest party and i'll merge without waiting for a review from an editorial WG codeowner.

### CODE REVIEW

File: eip155/caip10.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ Note that a given address cannot be assumed to work on all current and future ne
 
 ## Syntax
 
-Ethereum addresses were, historically, case-insensitive and normalized to use
-all-lowercase letters (`abcdef`) like most hexadecimal numeric types.  With the
-ratification of [EIP-55][], however, a particular normalization of lowercase- and
-uppercase- `abcdefABCDEF` characters was invented as an efficient form of
-checksum. See [EIP-55][] for specification.
+Ethereum addresses were, historically, case-insensitive and normalized to use all-lowercase letters (`abcdef`) like most hexadecimal numeric types.
+With the ratification of [EIP-55][], however, a particular normalization of lowercase- and uppercase- `abcdefABCDEF` characters was invented as an efficient form of checksum.
```
Comment (chris13524) [inline eip155/caip10.md:36]: Isn't this ERC-55 now?

File: eip155/caip10.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ Note that a given address cannot be assumed to work on all current and future ne
...
+Most implementations will still accept lowercase addresses but produce only checksum-case addresses, as many transaction-builders will validate not just against a regular expression but also against an EIP-55 checksum.
+Anywhere ["Postel's Law"][postel] can apply, implementers SHOULD produce checksum-case secure addresses (whether in CAIP-10 or native format), and SHOULD accept both checksum case and legacy lowercase addresses, except where the security concerns of the particular usecase outweigh interoperability.
```
Comment (chris13524) [inline eip155/caip10.md:38]: Nice still not making it mandatory (and this not a breaking change). But we still haven't guaranteed interoperability. Hopefully we can fix completely in the future

**Review by chris13524 — APPROVED** [review]

File: eip155/caip10.md (hunk 1 above)
Comment (bumblefudge) [reply eip155/caip10.md:36]: yes? "historically" meaning before EIP-55.  do i misunderstand?

File: eip155/caip10.md (hunk 2 above)
Comment (bumblefudge) [reply eip155/caip10.md:38]: well it's a final CAIP so literally we can't make a breaking normative change! what would fix completely look like? the whole point of this PR was to make status quo more explicit, any change to MUSTs and SHOULDs would have to be a new CAIP-number :D

File: eip155/caip10.md (hunk 1 above)
Comment (bumblefudge) [inline eip155/caip10.md:36]:
```suggestion
With the ratification of [ERC-55][], however, a particular normalization of lowercase- and uppercase- `abcdefABCDEF` characters was invented as an efficient form of checksum.
```

File: eip155/caip10.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ As the Ethereum namespace evolved, user-agents that connect to dapps through an
 [EIP-55]: https://eips.ethereum.org/EIPS/eip-55
```
Comment (bumblefudge) [inline eip155/caip10.md:104]:
```suggestion
[ERC-55]: https://eips.ethereum.org/EIPS/eip-55
```

**Review by obstropolos — APPROVED** [review]

**Pull request merged**